remote-api/src/main/java/org/alfresco/rest/api/model/rules/RuleSetLink.java

```diff
@@ -0,0 +1,39 @@
+/*
+ * #%L
+ * Alfresco Remote API
+ * %%
+ * Copyright (C) 2005 - 2022 Alfresco Software Limited
+ * %%
+ * This file is part of the Alfresco software.
+ * If the software was purchased under a paid Alfresco license, the terms of
+ * the paid license agreement will prevail.  Otherwise, the software is
+ * provided under the following open source license terms:
+ *
+ * Alfresco is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * Alfresco is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with Alfresco. If not, see <http://www.gnu.org/licenses/>.
+ * #L%
+ */
+
+package org.alfresco.rest.api.model.rules;
+
+public class RuleSetLink {
+    private String id;
```

Comment:
It would be useful to have some information about what this id is.  (I think in this PR it's always the node ref of the linked-to-folder)

Reply:
So initially I changed this `id` to `folderNodeId` but after a discussion with @mpichura ` we concluded that maybe we should re-think this. One other possible solution would be to add 2 different fields in this class, one for the folder node id and another for the ruleset node id. Or we could simply rename the existing one to `nodeId` and add a comment to provide further information. I wanted to know what others thing because this also impacts the api-explorer